crates/nu_plugin_query/src/query_web.rs

```diff
@@ -433,27 +451,36 @@ mod tests {
         <a href="https://example.com" target="_self">Example</a>
     "#;
 
+    fn null_spanned<T: ToOwned + ?Sized>(input: &T) -> Spanned<T::Owned> {
+        Spanned {
+            item: input.to_owned(),
+            span: Span { start: 0, end: 0 },
```

Comment:
Maybe this could be:
```suggestion
            span: Span::unknown(),
```

As we get to refactoring the span logic, having fewer places with explicit spans around is better. (you could also supersede your function with `IntoSpanned::into_spanned(Span::unknown())` but thats a question of taste)